incubator/vault/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Vault, a tool for managing secrets
 name: vault
-version: 0.4.3
+version: 0.4.4
```

Comment:
Because there are feature additions can you bump this to `0.5.0` for the version

Reply:
Fixed